Source/DataSources/StaticGroundGeometryPerMaterialBatch.js

```diff
@@ -110,12 +145,20 @@ define([
                     }
                 }
 
+                this.material = MaterialProperty.getValue(time, this.materialProperty, this.material);
+
                 primitive = new GroundPrimitive({
                     show : false,
                     asynchronous : true,
                     geometryInstances : geometries,
+                    appearance : new this.appearanceType({
+                        material : this.material,
+                        translucent : this.material.isTranslucent(),
+                        closed : this.closed
```

Comment:
`this.closed` isn't defined anywhere.

But `closed` should always be `true` for a ground primtiive

Reply:
Alas, an artifact of copy-pasta?
In my defense, the existing batches were kind of intimidating to read the first couple times...